src/main/java/chungus/Task.java

```diff
@@ -0,0 +1,405 @@
+package chungus;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashSet;
+
+import chungus.util.Pair;
+
+/**
+ * Represents a task.
+ */
+abstract class Task {
+    private String desc;
+    private boolean isDone;
+
+    protected HashSet<String> tags;
+
+    protected static final DateTimeFormatter DATETIME_FMT = DateTimeFormatter.ISO_LOCAL_DATE_TIME;
```

Comment:
Same here, perhaps the static attributes should be above the private final attributes according to coding standards